rosidl_typesupport_introspection_cpp/include/rosidl_typesupport_introspection_cpp/field_types.hpp

```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef __rosidl_typesupport_introspection_cpp__field_types__hpp__
-#define __rosidl_typesupport_introspection_cpp__field_types__hpp__
+#ifndef ROSIDL_TYPESUPPORT_INTROSPECTION_CPP__FIELD_TYPES_HPP_
+#define ROSIDL_TYPESUPPORT_INTROSPECTION_CPP__FIELD_TYPES_HPP_
```

Comment:
I think that technically it would be `ROSIDL_TYPESUPPORT_INTROSPECTION_CPP__ROSIDL_TYPESUPPORT_INTROSPECTION_CPP__FIELD_TYPES_HPP_`, but that rule of doing `<project>__<path>__<to>__<header>_<ext>_` maybe over kill since we almost always have the first element of the path be the name of the project.